e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -605,12 +605,12 @@ function verifyMessageAttachmentList(postId, isRhs, text) {
             and('have.css', 'height', '32px').
             and('have.css', 'width', '220px');
 
-        cy.findByPlaceholderText('Select an option...').
+        cy.findByPlaceholderText('Select an option...').scrollIntoView().
             and('have.css', 'height', '32px').
             and('have.css', 'width', '220px').
             and('have.css', 'padding-right', '30px');
 
-        return cy.findByPlaceholderText('Select an option...').invoke('attr', 'value').then((value) => {
+        return cy.findByPlaceholderText('Select an option...').scrollIntoView().invoke('attr', 'value').then((value) => {
```

Comment:
Got these failing
![Screen Shot 2020-03-12 at 10 16 59 AM](https://user-images.githubusercontent.com/5334504/76480869-ede84080-644a-11ea-9e5f-5514fc1fc740.png)

Reply:
I'm not getting these failures but I added force:true to clear. Hopefully that fixes it

Reply:
![Screen Shot 2020-03-12 at 9 40 48 AM](https://user-images.githubusercontent.com/487991/76544628-98f5fb80-6445-11ea-841b-99bdd587e8ae.png)

Reply:
Tested, all passed.